### PR TITLE
global: addition of js dependencies

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: invenio_deposit/jsonschemas/deposits/deposit-v1.0.0.json [hunk 1]
```diff
@@ -11,19 +11,6 @@
         "id": {
           "type": "string",
           "name": "id"
-        },
```
Comment (jirikuncar) [inline invenio_deposit/jsonschemas/deposits/deposit-v1.0.0.json]: Please remove this change. We are going to update deposit schema in other PR. Thanks

**Comment by drjova** [comment]
Depends on https://github.com/inveniosoftware/invenio-records-js/pull/7 and https://github.com/inveniosoftware/invenio-files-js/pull/4

**Comment by jirikuncar** [comment]
@drjova can you change commit message title from "deposition" to something like `global: addition of js dependencies`?

**Comment by drjova** [comment]
depends on https://github.com/inveniosoftware/invenio-records-files/pull/22

**Pull request closed without merging**